public/dark-theme.css

```diff
@@ -0,0 +1,74 @@
+.ant-input,
+.ant-input-number-input-wrap{
+  color :white !important;
+  background-color: #2c2f33 !important;
+}
+
+.ant-radio-button-wrapper{
+  background-color: #2c2f33 !important;
+  color: white !important;
+}
+
+.ant-form-item-label, input, .ant-form-item-label label, .ant-checkbox-wrapper, .ant-layout-footer{
+  color: white !important;
+}
+
+body{
+  background-color: #23272a !important;
+}
+
+form + div label {
+  margin-top: 10px;
+  color: white !important;
+  border-width: 1px !important;
+}
+
+form + div span {
+  border-width: 1px !important;
+}
+
+form + div {
+  margin-top: 10px;
+}
+
+.ant-radio-button-wrapper-checked{
+border-color: white !important;
+border-width: 2px !important;
```

Comment:
Nit: The indentation looks a little weird here and elsewhere in the file.

Reply:
reformatted in next commit